### ISSUE TITLE
Seek bar's aria-valuetext attribute should have times written out in words rather than numerical timestamp format

### ISSUE THREAD

**Issue by claudiah12** [videojs/video.js#6335]
## Description
The current time & duration in the aria-valuetext property of the seek bar is written as a timestamp, which in most cases screen readers will read as just numbers and punctuation, rather than something understandable as time. The time should written out in plain text like "x minutes, y seconds" for the aria value.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Turn on screen reader software
2. Navigate to the seek bar of a video
3. The screen reader will read the timestamp string as numbers and punctuation rather than an understandable time

### Expected
If seek bar is at 00:20/14:05, then screen reader announces 'twenty seconds of 14 minutes, 5 seconds

### Actual
If seek bar is at 00:20/14:05, then screen reader announces 'Zero colon zero colon twenty of fourteen colon one'.

**Comment by OwenEdwards** [comment]
@claudiah12 unfortunately, implementing a change like this is going to be *very* complicated to do correctly, because of the complexity of internationalizing/localizing plurals (see, for example, https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals).

You can look at https://github.com/OwenEdwards/video.js/pull/new/fix/improve-aria-text-of-progress-bar-for-screen-reader-users for a basic implementation in English, but even that doesn't handle plurals correctly.

**Comment by gkatsev** [comment]
I hadn't even considered how difficult it could be to do.

**Comment by gkatsev** [comment]
With Video.js 8, we're going to be dropping support for IE11, this means that Intl.PluralRules will be available in all the browsers we support, which should hopefully allow us to do a better job and do it more easily.
